ast/ast.go

```diff
@@ -1867,14 +1858,17 @@ type CreateProtoBundle struct {
 
 // AlterProtoBundle is ALTER PROTO BUNDLE statement node.
 //
-//	ALTER PROTO BUNDLE {{.Alteration | sql}}
+//	ALTER PROTO BUNDLE {{.Insert | sqlOpt}} {{.Update | sqlOpt}} {{.Delete | sqlOpt}}
```

Comment:
Are they actually restricted to this order? Also, is it not possible to specify the same operation multiple times?

Reply:
Don't permit reverse order.
```
spanner> ALTER PROTO BUNDLE DELETE(ProtoType) UPDATE(ProtoType);
ERROR: error on create op: rpc error: code = InvalidArgument desc = Error parsing Spanner DDL statement: ALTER PROTO BUNDLE DELETE(ProtoType) UPDATE(ProtoType) : Syntax error on line 1, column 38: Expecting 'EOF' but found 'UPDATE'
```
and don't permit duplicated.

```
spanner> ALTER PROTO BUNDLE UPDATE(ProtoType) UPDATE(ProtoType);
ERROR: error on create op: rpc error: code = InvalidArgument desc = Error parsing Spanner DDL statement: ALTER PROTO BUNDLE UPDATE(ProtoType) UPDATE(ProtoType) : Syntax error on line 1, column 38: Expecting 'EOF' but found 'UPDATE'
```